ipam/calico-ipam.go

```diff
@@ -222,18 +222,34 @@ func cmdDel(args *skel.CmdArgs) error {
 		return err
 	}
 
-	// Release the IP address by using the handle - which is workloadID.
-	workloadID, _, err := utils.GetIdentifiers(args)
+	// Release the IP address by using the handleID.
+	handleId, err := utils.GetHandleId(args)
 	if err != nil {
 		return err
 	}
 
-	logger := utils.CreateContextLogger(workloadID)
+	// Release the IP address by using the workloadID, to not leak IPs in existing installs.
+	workloadId, _, err := utils.GetIdentifiers(args)
+	if err != nil {
+		return err
+	}
+
+	logger := utils.CreateContextLogger(handleId)
```

Comment:
Let's have a logger with both workloadID and handle here.  It's super useful to be able to grep the kubelet log for the workload ID and see all the handles or vice versa.